source/conf.py

```diff
@@ -19,9 +19,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = u'perun-doc'
-copyright = u'2020, Hyperledger'
-author = u'Hyperledger'
+project = u'Perun Framework'
+copyright = u'2020, Perun Framework Contributors'
```

Comment:
2021.  
Regarding the contributors: Maybe we also need a `NOTICE` file here? @sebastianst

Reply:
Actually, the docs project started last year, so 2020 is okay IMO.

Reply:
Isn't the copyright year usually updated?

Reply:
It doesn't really matter that much. Some add years whenever there were substantive changes, i.e you'd procede from `2020` via `2020, 2021` to `2020--2022`...

Reply:
Thank you for your comments.

@choeppler I adjusted the copyright holder in the footer accordingly.